cdc/sink/codec/avro.go

```diff
@@ -466,6 +466,8 @@ func columnToAvroNativeData(col *model.Column) (interface{}, string, error) {
 			return handleUnsignedInt64()
 		}
 		return col.Value.(int64), "long", nil
+	case mysql.TypeFloat:
+		return col.Value.(float64), "float", nil
```

Comment:
Is it ok to use `float32` here, since float in TiDB/MySQL is 4 bytes

Reply:
OH, just ignore this comment, float is 8 bytes in TiDB

Reply:
If it were `float32`, the original implementation would have worked.